pyhap/accessory.py

```diff
@@ -135,12 +139,18 @@ def __init__(self, display_name, aid=None, mac=None, pincode=None,
             managed by an `AccessoryDriver` must have a pincode. The pincode has the
             format "xxx-xx-xxx", where x is a digit.
         @type pincode: bytearray
+
+        @param setup_id: Setup ID can be provided, although, per spec, should be random
+            every time the instance is started. If not provided on init, will be random.
+            4 digit string 0-9 A-Z
+        @type setup_id: str
         """
         self.display_name = display_name
         self.aid = aid
         self.mac = mac
         self.config_version = 2
         self.reachable = True
+        self.pincode = pincode
```

Comment:
This should be removed: see the pincode property/method

Reply:
ahh, sorry. missed it.

Reply:
ahh sorry, missed it.